### PR TITLE
[FEAT] handle qrcode import and join

### PR DESCRIPTION
**Pull request by gabrielbazan7**

### CODE REVIEW

File: src/navigation/wallet/screens/JoinMultisig.tsx [hunk 1]
```diff
@@ -153,28 +123,64 @@ const JoinMultisig = () => {
...
+                        params: {walletId: wallet.id, key},
+                      },
+                    },
+                  ],
+                }),
+              );
+            });
+          } else {
+            navigation.dispatch(
+              CommonActions.reset({
```
Comment (lanchana) [inline src/navigation/wallet/screens/JoinMultisig.tsx:160]: question: else will execute even when getStatus return err. is it intensional?

File: src/navigation/wallet/WalletStack.tsx [hunk 2]
```diff
@@ -342,6 +347,13 @@ const WalletStack = () => {
           name={WalletScreens.GLOBAL_SELECT}
           component={GlobalSelect}
         />
+        <Wallet.Screen
```
Comment (lanchana) [inline src/navigation/wallet/WalletStack.tsx:350]: question: how to test this?

File: src/store/scan/scan.effects.ts [hunk 3]
```diff
@@ -701,3 +709,43 @@ const handlePlainAddress =
...
+    },
+  });
+};
+
+const goToJoinWallet =
+  (data: string): Effect<void> =>
+  (dispatch, getState) => {
+    console.log('Incoming-data (redirect): Code to join to a multisig wallet');
+    const keys = Object.values(getState().WALLET.keys);
+    if (keys.length === 0) {
```
Comment (lanchana) [inline src/store/scan/scan.effects.ts:727]: nit: can be simplified to `if (!keys.length) {`

File: src/navigation/wallet/WalletStack.tsx (hunk 2 above)
Comment (gabrielbazan7) [reply src/navigation/wallet/WalletStack.tsx:350]: you need to have more than 2 backup keys and use the general scanner for the invitation code

File: src/navigation/wallet/screens/JoinMultisig.tsx (hunk 1 above)
Comment (gabrielbazan7) [reply src/navigation/wallet/screens/JoinMultisig.tsx]: Let's ignore the error in this case and just navigate to the key view

**Review by lanchana — COMMENTED** [review]
ACK!

**Review by JohnathanWhite — APPROVED** [review]

**Pull request merged**